specs/person_spec.rb

```diff
@@ -0,0 +1,20 @@
+relative_require '../person'
```

Comment:
- [ ] the tests are not running correctly because in this line you are using `relative_require` however the keyword you should use is `require_relative`, what you can do to avoid this kind of mistake is to enable an autofill extension in VS code for example.